### PR TITLE
[Tizen] Disable nnfw by default @open sesame 09/21 13:35

### PR DESCRIPTION
**Pull request by gichan-jang**
Disable nnfw by default
 - To avoid armcl dependency problem on arm arch.

### CODE REVIEW

**Review by wooksong — COMMENTED** [review]
```api``` requires ```libnnstreamer-edge.so.0.2.4``` but PPA provides ```libnnstreamer-edge.so.0.2.5``` after https://github.com/nnstreamer/nnstreamer-edge/commit/9f898ebb46e91c9592b467ec1ad64cbaa8362987.

FYI,
```bash
$ sudo ldconfig -p | grep edge
        libnnstreamer-edge.so.0.2.5 (libc6,x86-64) => /lib/x86_64-linux-gnu/libnnstreamer-edge.so.0.2.5
        libnnstreamer-edge.so (libc6,x86-64) => /lib/x86_64-linux-gnu/libnnstreamer-edge.so

$ apt policy nnstreamer-edge
nnstreamer-edge:
  Installed: 0.2.5.0-0~202309190236~ubuntu20.04.1
  Candidate: 0.2.5.0-0~202309190236~ubuntu20.04.1
  Version table:
 *** 0.2.5.0-0~202309190236~ubuntu20.04.1 500
        500 http://ppa.launchpad.net/nnstreamer/ppa/ubuntu focal/main amd64 Packages
        100 /var/lib/dpkg/status
```

File: packaging/machine-learning-api.spec [hunk 1]
```diff
@@ -12,7 +12,7 @@
 %define		tensorflow_lite_support	1
 %define		tensorflow2_lite_support 1
 %define		tensorflow2_gpu_delegate_support 1
-%define		nnfw_support 1
+%define		nnfw_support 0
```
Comment (myungjoo) [inline packaging/machine-learning-api.spec:15]: Can you avoid error simply by removing:

https://github.com/nnstreamer/api/blob/80f62746c1d37cf0ca0d9f8d6c67a44bbfb38c20/packaging/machine-learning-api.spec#L141-L142

Comment (gichan-jang) [reply packaging/machine-learning-api.spec]: Thanks, Updated.
I checked that OneRT also removed armcl backend onTizen.

**Review by songgot — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**